debian/control

```diff
@@ -26,6 +26,7 @@ Architecture: all
 Depends: python3-faucet (= FAUCET_VERSION),
          python3-netaddr,
          python3-networkx,
+         python3-prometheus-client (>= 0.7.1), python3-prometheus-client (<< 0.7.2),
```

Comment:
I think you can just do = for exact match

Reply:
Done